workflows/dev_deploy.yml

```
@@ -0,0 +1,11 @@
+name: UMC Dev CI/CD
+
+on:
+  pull_request:
+    types: [closed]
+  workflow_dispatch: # (2).수동 실행도 가능하도록
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)